ch_backup/config.py

```diff
@@ -154,7 +154,7 @@ def _as_seconds(t: str) -> int:
     },
     "loguru": {
         "formatters": {
-            "ch-backup": "{time:YYYY-MM-DD H:m:s,SSS} {process.name:11} {process.id:5} [{level:8}] {extra[logger_name]}: {message}",
+            "ch-backup": "{time:YYYY-MM-DD HH:mm:ss,SSS} {process.name:11} {process.id:5} [{level:8}] {extra[logger_name]}: {message}",
```

Comment:
Looks unrelated to the memory usage fix.

Reply:
It looks like the branch was forked not from main, because this changes already in main.